rotkehlchen/chain/ethereum/modules/makerdao/constants.py

```diff
@@ -10,6 +10,9 @@
 CPT_VAULT = 'makerdao vault'
 CPT_DSR = 'makerdao dsr'
 CPT_MIGRATION = 'makerdao migration'
+CPT_SDAI = 'sDAI'
 
 MAKERDAO_LABEL = 'Makerdao'
 MAKERDAO_ICON = 'makerdao.svg'
+SDAI_LABEL = 'sDAI contract'
```

Comment:
This should have also just been sDAI probably. But we can change it later if needed. Leave it as is for now and see if it looks okay.